### ISSUE TITLE
🐛 [firebase auth] Firebase auth breaks pod installl

### ISSUE THREAD

**Issue by safield** [FirebaseExtended/flutterfire#8141]
I am recieving the following error when running pod install in my flutter app ios folder.

```
[!] CocoaPods could not find compatible versions for pod "Firebase/Auth":
  In Podfile:
    Firebase/Auth

    firebase_auth (from `.symlinks/plugins/firebase_auth/ios`) was resolved to 3.3.7, which depends on
      Firebase/Auth (= 8.11.0)

Specs satisfying the `Firebase/Auth, Firebase/Auth (= 8.11.0)` dependency were found, but they required a higher minimum deployment target.
```

Flutter doctor output
```

[✓] Flutter (Channel stable, 2.8.1, on macOS 12.1 21C52 darwin-arm, locale en-CA)
    • Flutter version 2.8.1 at /Users/scott/.flutter-sdk
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 77d935af4d (10 weeks ago), 2021-12-16 08:37:33 -0800
    • Engine revision 890a5fca2e
    • Dart version 2.15.1

[✗] Android toolchain - develop for Android devices
    ✗ Unable to locate Android SDK.
      Install Android Studio from: https://developer.android.com/studio/index.html
      On first launch it will assist you in installing the Android SDK components.
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).
      If the Android SDK has been installed to a custom location, please use
      `flutter config --android-sdk` to update to that location.


[✓] Xcode - develop for iOS and macOS (Xcode 13.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[!] Android Studio
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    ✗ Unable to find bundled Java version.
    • Try updating or re-installing Android Studio.

[✓] VS Code (version 1.64.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.34.0

[✓] Connected device (2 available)
    • dev iphone (mobile) • f709b321adae72b7bb72f2ca9f4eb87aec329318 • ios            • iOS 15.3.1 19D52
    • Chrome (web)                • chrome                                   • web-javascript • Google Chrome 98.0.4758.102
    ! Error:dev iphone is busy: Preparing Scott's dev iphone for development. Xcode will continue when dev iphone is finished. (code -10)
```

**Comment by darshankawar** [comment]
> [!] CocoaPods could not find compatible versions for pod "Firebase/Auth":
>   In Podfile:
>     Firebase/Auth
> 
>     firebase_auth (from `.symlinks/plugins/firebase_auth/ios`) was resolved to 3.3.7, which depends on
>       Firebase/Auth (= 8.11.0)
> 
> Specs satisfying the `Firebase/Auth, Firebase/Auth (= 8.11.0)` dependency were found, but they required a higher minimum deployment target.

As the message indicates, the firebase_auth version you are using probably requires higher minimum deployment target, so please set the target as 10.0 in Xcode if you haven't done so already. See this document section for your reference: https://firebase.flutter.dev/docs/manual-installation/ios#ios-supported-versions

Also, the plugin version you are using may require the pod to use `Firebase/Auth (= 8.11.0)`. So you can delete the podfile.lock and regenerate it which should pickup updated pod version.

**Comment by nj9666** [comment]
I use firebase_auth: ^3.3.10 in pubspec.yaml
when I run in ios than I got this issue 


```
CDN: trunk Relative path: Specs[/0/3/5/Firebase/8.5.0/Firebase.podspec.json]() exists! Returning local because checking is only performed in repo update
    [!] CocoaPods could not find compatible versions for pod "Firebase[/Auth]()":
      In snapshot (Podfile.lock):
        Firebase[/Auth]() (= 8.5.0)
      In Podfile:
        firebase_auth (from `.symlinks[/plugins/firebase_auth/ios]()`) was resolved to 3.3.10, which depends on
          Firebase[/Auth]() (= 8.11.0)
    Specs satisfying the `Firebase[/Auth]() (= 8.5.0), Firebase[/Auth]() (= 8.11.0)` dependency were found, but they required a higher minimum deployment target.
```
